{{cookiecutter.project_slug}}/clients/web/vue3/tests/e2e/specs/test-password-reset.cy.ts

```diff
@@ -0,0 +1,15 @@
+describe('Tests password reset workflow', () => {
+  it('Login page has link to reset password', () => {
+    cy.visit('/login')
+    cy.contains('[data-cy=password-reset]', 'Forgot password?')
+  })
+
+  it('Submitting email doesnt throw error', () => {
+    cy.visit('/login')
+    cy.contains('[data-cy=password-reset]', 'Forgot password?').click()
+    cy.url().should('include', '/password/request-reset/')
+    cy.get('[data-cy=email]').type(Cypress.env('TEST_USER_EMAIL'))
+    cy.contains('[data-cy=submit]', 'Request Password Reset').click()
+    cy.get('[data-cy=submit-success]').should('contain.text', 'Your request has been submitted')
+  })
+})
```

Comment:
Adding a basic test here to help avoid this breaking again.
If the backend has an error, the frontend won't show the `"Your request has been submitted..."` message